specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,456 @@
+## Reference Implementation
+
+- Common [type definitions](https://github.com/alpha-fi/i-am-human/tree/master/contracts/sbt) (events, traits).
+- https://github.com/alpha-fi/i-am-human/tree/master/contracts/soulbound
+
+## Example Flows
+
+```mermaid
+sequenceDiagram
+    actor Alice
+    actor Issuer1 as SBT_1 Issuer
+
+    participant SBT1 as SBT_1 Contract
+    participant SBT_Registry
+
+    Issuer1->>SBT1: sbt_mint(alice, 1, metadata)
+    activate SBT1
+    SBT1-)SBT_Registry: sbt_mint([[alice, metadata])
+    SBT_Registry->>SBT_Registry: emit Mint(SBT_1_Contract, alice, [238])
+    SBT_Registry-)SBT1: [238]
+    deactivate SBT1
+
+    Note over Alice,SBT_Registry: now Alice can query registry to check her SBT
+
+    Alice-->>SBT_Registry: sbt(SBT_1_Contract, 238)
+    SBT_Registry-->>Alice: {token: 238, owner: alice, metadata}
+
+    Note over Alice,SBT_Registry: Alice participates in other community and applies for another SBT.<br/> She will get 2 SBTs. However she uses a different wallet: alice2.
+
+    participant SBT2 as SBT_2 Contract
+    actor Issuer2 as SBT_2 Issuer
+
+    Issuer2->>SBT2: sbt_mint_multi([[alice2, metadata2], [alice2, metadata3]])
+    activate SBT2
+    SBT2-)SBT_Registry: sbt_mint([[alice2, metadata2], [alice2, metadata3]])
+    SBT_Registry->>SBT_Registry: emit Mint(SBT_2_Contract, alice2, [7991, 17992])
+    SBT_Registry-)SBT2: [7991, 1992]
+    deactivate SBT2
+
+    Alice-->>SBT_Registry: sbt(SBT_2_Contract, 7991)
+    SBT_Registry-->>Alice: {token: 7991, owner: alice2, metadata: metadata2}
+
+    Note over Alice,SBT_Registry: After a while Alice decides to merge her `alice2` account into `alice2`
+    Alice->>SBT_Registry: sbt_soul_transfer(alice) --accountId alice2
+
+    Alice-->>+SBT_Registry: sbt_tokens_by_owner(alice2)
+    SBT_Registry-->>-Alice: []
+
+    Alice-->>+SBT_Registry: sbt_tokens_by_owner(alice)
+    SBT_Registry-->>-Alice: [[SBT_1_Contract, [238]], [SBT_2_Contract, [7991, 7992]]]}
+```
+
+## Consequences
+
+### Positive
+
+- Template and set of guidelines for creating SBT tokens.
+- Ability to create SBT aggregators.
+- Ability to use SBT as a primitive to model non KYC identity, badges, certificates etc...
+- SBT can be further used for "lego" protocols, like: Proof of Humanity (discussed for NDC Governance), undercollateralized lending, role based authentication systems, innovative economic and social applications...
+- Standard recover-ability mechanism.
+- SBT are considered as a basic primitive for Decentralized Societies.
+- new way to implement Sybil attack resistance.
+
+### Neutral
+
+- The API partially follows the NEP-171 (NFT) standard. The proposed design is to have native SBT API and also support NFT based queries.
+  NOTE: we can decide to use `nft_` prefix whenever possible.
+
+### Negative
+
+- new set of events to be handled by the indexer and wallets.
+- complexity of integration with a registry: all SBT related transactions must go through Registry.
+
+## Considerations
```

Comment:
This part should be removed before the final approval, correct?

Reply:
not necessary. Some consideration can be left. But definitely this must be cleared out and documented with the decisions.